client/Makefile

```diff
@@ -15,7 +15,7 @@ GO_PACKAGES_REPO_PATH = $(addprefix $(repo_path)/,$(GO_PACKAGES))
 GOFMT = gofmt -e -l -s
 GOTEST = go test --cover --race -v
 
-VERSION := $(shell git describe --tags --abbrev=0 2>/dev/null)+$(shell git rev-parse --short HEAD)
+VERSION ?= $(shell git rev-parse --short HEAD)
```

Comment:
So, sorry for all the noise -- this is the only change for this PR, to align with what deis.io looks for in bintray:

```
 ~/code/deis.io/deis-cli  ⑂ update-deiscli-bin-name    ./install-v2-alpha.sh
Downloading deis-git-5e08df4-darwin-amd64 from Bintray...
```